core/src/main/java/org/apache/gravitino/Configs.java

```diff
@@ -389,4 +389,12 @@ private Configs() {}
           .version(ConfigConstants.VERSION_1_0_0)
           .booleanConf()
           .createWithDefault(true);
+
+  // Provider name for cache
+  public static final ConfigEntry<String> CACHE_PROVIDER =
+      new ConfigBuilder("gravitino.cache.provider")
+          .doc("Name of the cache provider to use.")
+          .version(ConfigConstants.VERSION_1_0_0)
+          .stringConf()
+          .createWithDefault("Caffeine");
```

Comment:
Please change to "caffeine", do not capitalize, which will easily lead to mis-configuraition.